casper-event-standard/src/lib.rs

```diff
@@ -70,6 +70,10 @@ pub const EVENTS_DICT: &str = "__events";
 pub const EVENTS_LENGTH: &str = "__events_length";
 /// The key under which the event schemas are stored.
 pub const EVENTS_SCHEMA: &str = "__events_schema";
+/// The key under which the ces version is stored.
+pub const CES_VERSION_KEY: &str = "ces_version";
```

Comment:
I think the naming of all keys should be the same. Change it to` __events_version` in eg or rename other strings. The variable itself could also be `CES_VERSION`.